k8s/vault.yaml

```diff
@@ -1,18 +1,32 @@
+apiVersion: extensions/v1beta1
```

Comment:
Please include documentation to the ingress API. 

@sethvargo do you have Apache headers in your manifests?